src/ReportGenerator.Core/Reporting/Builders/Rendering/HtmlRenderer.cs

```diff
@@ -686,11 +687,11 @@ public void MetricsTable(Class @class)
                             WebUtility.HtmlEncode(methodMetric.FullName),
                             fileIndex,
                             methodMetric.Line,
-                            WebUtility.HtmlEncode(methodMetric.ShortName));
+                            WebUtility.HtmlEncode($"File {fileIndex} => " + methodMetric.ShortName));
                     }
                     else
                     {
-                        this.reportTextWriter.Write("<td title=\"{0}\">{1}</td>", WebUtility.HtmlEncode(methodMetric.FullName), WebUtility.HtmlEncode(methodMetric.ShortName));
+                        this.reportTextWriter.Write("<td title=\"{0}\">{1}</td>", WebUtility.HtmlEncode(methodMetric.FullName), WebUtility.HtmlEncode(file.Path + " => " + methodMetric.ShortName));
```

Comment:
Add file index before the method name, so that we know which file is associated with this method instance. This is to handle the scenario that multiple files may be invoke for the same method, and there are multiple method instances listed on the class coverage page.